src/redisearch_rs/inverted_index/src/lib.rs

```diff
@@ -37,18 +37,21 @@ impl From<Delta> for usize {
 /// Represents a numeric value in an index record.
 /// cbindgen:field-names=[value]
 #[allow(rustdoc::broken_intra_doc_links)] // The field rename above breaks the intra-doc link
+#[derive(Clone, Copy)]
 #[repr(C)]
 pub struct RSNumericRecord(pub f64);
 
 /// Represents the encoded offsets of a term in a document. You can read the offsets by iterating
 /// over it with RSOffsetVector_Iterator
+#[derive(Clone, Copy)]
 #[repr(C)]
 pub struct RSOffsetVector {
     pub data: *mut c_char,
     pub len: u32,
 }
```

Comment:
Let's not derive `Copy` for types that contain raw pointers, it's a recipe for bugs down the line.

Reply:
We are doing `ManuallyDrop` instead on the `union` types.